src/components/OnBoarding/OnBoarding.module.scss

```diff
@@ -0,0 +1,132 @@
+@import '../../styles/colors.scss';
+@import '../../styles/variables.scss';
+
+.OnBoarding {
+ 
+  .overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-image: linear-gradient(to bottom, #6c7694, #353c52);
+    z-index: 2;
+  }
+
+  .onBoarding {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 2;
+    background-color: $color-dark-blue-100;
+    box-shadow: 0 2px 16px 0 rgba(0, 0, 0, 0.16);
+    opacity: 1;
+
+    @media (min-width: $mobile-width + 1px) {
+      border-radius: 6px;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      padding-left:56px; 
+    padding-right:56px; 
+    }
+    @media (max-width: $mobile-width) {
+      height: 100%;
+      overflow-y: auto;
+    }
+  }
+
+  .progressBar {
+    display: flex;
+    flex-direction: row;
+    .bulletActive {
+      width: 5px;
+      height: 5px;
```

Comment:
you can merge css properties between active and inactive. Just define a default `bullet` class, and an `bulletActive` class which overrides the color